templates/taxi/car_detail.html

```diff
@@ -19,4 +13,18 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  
+  {% if user in car.drivers.all %}
```

Comment:
The condition `{% if user in car.drivers.all %}` might not work as expected because `car.drivers.all` is a queryset, and `user` is an object. Consider using `{% if user in car.drivers.all %}` with `.all()` replaced by `.all()` to ensure it returns a list, or use a method like `.filter()` to check for membership.